scanner/indexer/options.go

```diff
@@ -26,6 +27,12 @@ func WithAuth(auth authn.Authenticator) Option {
 	}
 }
 
+func InsecureSkipTLSVerify(insecure bool) Option {
```

Comment:
I guess you can say "InsecureSkipTLSVerify" specifies if TLS verification should be skipped when reaching out to the registry.